etcdmain/gateway.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/coreos/etcd/pkg/transport"
 	"github.com/coreos/etcd/proxy/tcpproxy"
 	"github.com/spf13/cobra"
+	"strings"
```

Comment:
standard library packages (e.g., net, os, strings) should be group together at the beginning of the import